scripts/pyqt5_to_pyqt6/pyqt5_to_pyqt6.py

```diff
@@ -173,6 +192,10 @@ def get_class_enums(item):
         if inspect.isclass(value) and type(value).__name__ == 'EnumType':
             for e in value:
                 qt_enums[(item.__name__, e.name)] = f"{value.__name__}"
+
+            for _extra_value in extra_members.get((item.__name__, value.__name__), []):
+                qt_enums[(item.__name__, _extra_value)] = f"{value.__name__}"
+
```

Comment:
```suggestion
for ekey, evalue in value.__dict__.items():
+                if (isinstance(evalue, value)):
+                    qt_enums[(item.__name__, ekey)] = f"{value.__name__}"
```

This should solve things for all extra_members like enum

Reply:
Thanks, that's indeed much better!